filebeat/module/mongodb/log/_meta/fields.yml

```diff
@@ -0,0 +1,26 @@
+- name: log
+  type: group
+  description: >
+      Contains fields from MongoDB logs.
+  fields:
+  - name: timestamp
+    description: >
+        Timestamp of message
+  - name: severity
+    description: >
+        Severity level of message
+    example: I
+    type: keyword
```

Comment:
I came up with field names based on MongoDB logging guide: https://docs.mongodb.com/manual/reference/log-messages/ It might provide more context on the subject.
However, I agree that it is a bit weird in MongoDB. If you look at the section "Severity", you can see the taxonomy of log levels. So I am not sure why it's named severity.
Does it cause problem in ECS?

Reply:
I think we can keep it as is as it's under the mongodb namespace. In case we map this to ECS later it would end up in `log.level` from my perspective. No need to solve this now.